.circleci/config.yml

```diff
@@ -67,19 +158,149 @@ jobs:
+      - run: mkdir -p ./dist/macos
+      - run: sh -x ./scripts/cp_artifacts.sh release ./dist/macos
+      - persist_to_workspace:
+          root: ./dist
+          paths:
+            - ./macos/
+      # - run:
+      #     name: decode Apple Certificates
+      #     command: base64 -D -o Certificates.p12 <<< $applecertificates
+      # - run:
+      #     name: make Provisioning Profiles directory
+      #     command: mkdir -pv ~/Library/MobileDevice/Provisioning\ Profiles/
+      # - run:
+      #     name: decode Provisioning Profiles
+      #     command: base64 -D -o ~/Library/MobileDevice/Provisioning\ Profiles/App_Store.mobileprovision <<< $App_Store_Profile
+
+  store:
+    executor: wine-chrome
+    steps:
+      - attach_workspace:
+          at: ./dist
       - store_artifacts:
-          path: /tmp/artifacts/
+          path: ./dist
           destination: packages
-      - run: ./node_modules/.bin/build-storybook -c src/.storybook -o storybook
-      - store_artifacts:
-          path: storybook/
-          destination: storybook
+
 
 workflows:
   version: 2
-  build_and_test:
+  test_and_build:
     jobs:
       - check
-      - build:
+      - build-linux:
+          requires:
+             - check
+      - build-win-no-installer:
+          requires:
+             - check
+          filters:
+            branches:
+              ignore:
+                - /^(v|V)\.\d+\.\d+(\.\d+)?(-(rc|RC)-.*/)?/
+                # todo: once working remove this one
+                - mac_build
+      - build-mac-no-dmg:
+          requires:
+             - check
+          filters:
+            branches:
+              ignore:
+                - /^(v|V)\.\d+\.\d+(\.\d+)?(-(rc|RC)-.*/)?/
+                # todo: once working remove this one
+                - mac_build
+
+      - msi_installer:
           requires:
             - check
+          filters:
+            branches:
+              only: 
+              # only for release and release candidates
+              # v.XX.YY
+              # v.XX.YY.ZZ
+              # v.XX.YY.ZZ-rc-something
+                - /^(v|V)\.\d+\.\d+(\.\d+)?(-(rc|RC)-.*/)?/
+                # todo: once working remove this one
+                - mac_build
+
+      - mac_installer:
+          requires:
+            - check
+          context: codesign-certificates
+          filters:
+            branches:
+              only:
+                - /^(v|V)\.\d+\.\d+(\.\d+)?(-(rc|RC)-.*/)?/
+                # todo: once working remove this one
+                - mac_build
+      - store:
+          # for master builds
+          requires:
+            - build-linux
+            - build-win-no-installer
+            - build-mac-no-dmg
+          filters:
+            branches:
+              ignore:
+                - /^(v|V)\.\d+\.\d+(\.\d+)?(-(rc|RC)-.*/)?/
+                # todo: once working remove this one
+                - mac_build
+      - store:
```

Comment:
can you rename this to another better name? store what?

Reply:
+1

Reply:
would `store_artifacts` be good enough? currently this job gets results from the rest of the jobs and publish them on the artifacts section